### PR TITLE
(Delegator) Migrate delegator codebase (root -> node)

### PR DESCRIPTION
**Pull request by nick-bisonai**
# Description

Removal of pre-existing code will be done after it has been confirmed to work as expected

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

### CODE REVIEW

**Comment by nick-bisonai** [comment]
@coderabbitai resolve

File: node/migrations/delegator/000002_update_tx_table.dopwn.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
+DO $$
```
Comment (Intizar-T) [inline node/migrations/delegator/000002_update_tx_table.dopwn.sql:1]: a typo in the file name `dopwn` -> `down`

**Comment by Intizar-T** [comment]
it seems like `delegator` will use the `node` db, right? If so, I'm guessing we should move all the data from `delegator` db tables to the new tables in the `node` db?

**Review by Intizar-T — APPROVED** [review]
lgtm!

**Comment by nick-bisonai** [comment]
> it seems like `delegator` will use the `node` db, right? If so, I'm guessing we should move all the data from `delegator` db tables to the new tables in the `node` db?

no it won't, it'll keep using its own db 😅

**Pull request merged**